File: AircraftWar/src/main/scala/edu/hitsz/bullet/HeroBullet.scala
package edu.hitsz.bullet

import edu.hitsz.animate.AnimateContainer
import edu.hitsz.application.ImageResource
import edu.hitsz.basic.PositionType.Position
import edu.hitsz.basic.Vec2Double

/**
 * @author chiro2001
 */
class HeroBullet(posInit: Position, animateContainer: AnimateContainer[Vec2Double], power: Int)
  extends AbstractBullet(posInit, animateContainer, power)
  with ImageResource {
  override def getImageCachedPath = "images/bullet_hero.png"
}

File: src/main/java/edu/hitsz/bullet/HeroBullet.java
package edu.hitsz.bullet;

import edu.hitsz.animate.AnimateContainer;
import edu.hitsz.vector.Vec2;

/**
 * @author hitsz
 */
public class HeroBullet extends BaseBullet {

    public HeroBullet(Vec2 posInit, AnimateContainer animateContainer, int power) {
        super(posInit, animateContainer, power);
    }
}
